.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,8 @@ assignees: ''
 
 ---
 
+<!-- We've collected some common issue solutions in https://taichi.readthedocs.io/en/stable/install.html#troubleshooting. Make sure you've check them out first. Hopefully they could address your problem. -->
+
```

Comment:
Off-topic change. To make sure users have check out our doc before submit an duplicate issue.